src/compile-middleware.ts

```diff
@@ -144,6 +144,7 @@ export function browserNeedsCompilation(userAgent: string): boolean {
   const supportsES2015 = (browser.name === 'Chrome' && majorVersion >= 49) ||
       (browser.name === 'Chromium' && majorVersion >= 49) ||
       (browser.name === 'OPR' && majorVersion >= 36) ||
+      (browser.name === 'Vivaldi' && majorVersion >= 1) ||
```

Comment:
Vivaldi has used Chromium 49 since the [1.0 release](https://vivaldi.com/blog/snapshots/vivaldi-finale-1-0/), but there were snapshot builds before that which had support. Here's the UA string for the [earliest snapshot which used Chromium 49](https://vivaldi.com/blog/snapshots/snapshot-1-0-420-4-search-suggestions-and-further-bug-fixing/):

```
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.91 Safari/537.36 Vivaldi/1.0.420.4
```